### ISSUE TITLE
collection structure

### ISSUE THREAD

**Issue by mathias-goebel** [dracor-org/dracor-api#38]
@cmil you came up with notable thoughts on the current structure of the collections in the database. as far as i remember this should lead to something like the following:
```
db
└── data
    └── dracor
        ├── metrics
        │   ├── ger
        │   ├── rus
        │   ├── shake
        │   └── swe
        ├── rdf
        │   ├── ger
        │   ├── rus
        │   ├── shake
        │   └── swe
        └── tei
            ├── ger
            ├── rus
            ├── shake
            └── swe
```
is this correct and if not, may be you can provide a better sample?

**Comment by lehkost** [comment]
Thanks a lot!! – And while we haven't released our 1.0 yet, we can still change anything at any time. So feel free to consistently rename stats → metrics.

**Comment by ingoboerner** [comment]
@mathias-goebel suggested to store the rdf of the plays also in separate files, I changed the function to generate an `<rdf:RDF>` node for each play, therefore it should be possible to store them accordingly as separate files in the for-loop.
